pkg/server/endpoints/node/handler.go

```diff
@@ -219,7 +218,66 @@ func (h *Handler) FetchX509SVID(server node.Node_FetchX509SVIDServer) (err error
 }
 
 func (h *Handler) FetchJWTSVID(ctx context.Context, req *node.FetchJWTSVIDRequest) (*node.FetchJWTSVIDResponse, error) {
-	return nil, errors.New("not implemented")
+	peerCert, err := h.getCertFromCtx(ctx)
+	if err != nil {
+		h.c.Log.Error(err)
+		return nil, errors.New("node SVID is required for this request")
```

Comment:
nit: this would be clearer if it was something like `client SVID is required for this request` or `authentication is required for this request`

Reply:
done.